next-cloudinary/src/components/CldUploadButton/CldUploadButton.tsx

```diff
@@ -33,13 +33,21 @@ const CldUploadButton = ({
   ...props
 }: CldUploadButtonProps) => {
 
+  const actionProps = Object.keys(props)
```

Comment:
i think we can just pass through all of the `props` at this point and avoid filtering them down here

this will allow the Button to always inherit any available options on the UploadWidget without having to add it in both places, unless you're doing this because of the TS issue you mentioned before